Icons property does not exist on class pluginDefinition

## Type of report

Bug

Creating a custom plugin as described in the [tutorial](https://ckeditor.com/docs/ckeditor4/latest/guide/plugin_sdk_sample_1.html#plugin-source-code), shows that an icons property can be defined but no such property is mentioned in the [pluginDefintion class](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_pluginDefinition.html). So, when adding a icon property in the custom plugin, typescript flags the following error: 

```
error TS2345: Argument of type '{ icons: string; init: (editor: editor) => void; }' is not assignable to parameter of type 'pluginDefinition'.
  Object literal may only specify known properties, and 'icons' does not exist in type 'pluginDefinition'.
```

## Provide detailed reproduction steps (if any)

1. Create a custom plugin with icons specified (same as tutorial [here](https://ckeditor.com/docs/ckeditor4/latest/guide/plugin_sdk_sample_1.html#plugin-source-code))
2. Compile the typescript file having the plugin definition

### Expected result

No error should be raised complaining about missing icons property.

### Actual result

Error is flagged that icons does not exist in type pluginDefintion.

## Other details

* Browser: Chrome v72.0.3626.109
* OS: Ubuntu
* CKEditor version: 4.9.2
* Installed CKEditor plugins: sharedspace

--- a/core/plugindefinition.js
+++ b/core/plugindefinition.js
@@ -177,12 +177,14 @@
  */
 
 /**
- * The list of icons available for this plugin. These files are stored inside
+ * The list of icon files available for this plugin. These files are stored inside
  * the `icons` directory in the plugin directory, follow the name
  * pattern of `name.png`.
  *
+ * Usage:
+ *
  *		CKEDITOR.plugins.add( 'sample', {
- *			icons: 'first.png,second.png'
+ *			icons: 'first,second'
  *		} );
  *
  * @property {String} [icons]

--- a/core/plugindefinition.js
+++ b/core/plugindefinition.js
@@ -177,15 +177,15 @@
  */
 
 /**
- * The list of icon files available for this plugin. These files are stored inside
+ * The list of icon files registered by this plugin. These files are stored inside
  * the `icons` directory in the plugin directory, follow the name
  * pattern of `name.png`.
  *
- * Usage:
- *
- *		CKEDITOR.plugins.add( 'sample', {
- *			icons: 'first,second'
- *		} );
+ * ```javascript
+ *	CKEDITOR.plugins.add( 'sample', {
+ *		icons: 'first,second'
+ *	} );
+ * ```
  *
  * @property {String} [icons]
  */

--- a/core/plugindefinition.js
+++ b/core/plugindefinition.js
@@ -175,3 +175,15 @@
  * @since 4.2
  * @property {Boolean} hidpi
  */
+
+/**
+ * The list of icons available for this plugin. These files are stored inside
+ * the `icons` directory in the plugin directory, follow the name
+ * pattern of `name.png`.
+ *
+ *		CKEDITOR.plugins.add( 'sample', {
+ *			icons: 'first.png,second.png'
+ *		} );
+ *
+ * @property {String} [icons]
+ */

